src/components/Amount/Amount.tsx

```diff
@@ -14,6 +14,7 @@ export type AmountProps = {
   abbreviated?: boolean
   truncateLargeNumbers?: boolean
   maximumFractionDigits?: number
+  noSpace?: boolean
```

Comment:
Would have preferred not to add another prop, but touching the actual implementation to remove the extra spaces is a no go as it's used in many places throughout the app.